app/controllers/request-to-go-live/organisation-name/post.controller.js

```diff
@@ -0,0 +1,11 @@
+'use strict'
+
+// Local dependencies
+const goLiveStageToNextPagePath = require('../go-live-stage-to-next-page-path')
+
+module.exports = (req, res) => {
+  res.redirect(
+    303,
+    goLiveStageToNextPagePath[req.service.currentGoLiveStage].replace(':externalServiceId', req.service.externalId)
```

Comment:
We have a util that can do this, but using a native JS func seems legit too

Util —> https://github.com/alphagov/pay-selfservice/blob/master/app/utils/replace_params_in_path.js

Reply:
Yes, thanks, we think if we have multiple params it's nice to use the custom util, but we think it is better not to depend on custom code in our case